### PR TITLE
Broken Import Fix

### PR DESCRIPTION
**Pull request by astruxie**
When I tried to run develop the imports for the cli controller were broken, so I fixed those

I'm not sure what the printColor module is but I simply commented out that import line because it doesn't seem to be used by anything at the moment. 

I also added cmd2 to the requirements text

xoxo  💋

### CODE REVIEW

File: controller/cliController.py [hunk 1]
```diff
@@ -7,14 +7,14 @@
 import pyfiglet
 import cmd2
 # Local Imports
-import UMLClass
-import attributes
-import relationship
-import parameter
+import model.UMLClass
```
Comment (Spyder-Monkey) [inline controller/cliController.py:10]: for each of the model classes, do _import model.<className> as <className>_

**Review by Spyder-Monkey — CHANGES_REQUESTED** [review]
Fix the import statements in cliController to avoid import errors

File: controller/cliController.py (hunk 1 above)
Comment (astruxie) [reply controller/cliController.py:10]: okay

**Review by Spyder-Monkey — APPROVED** [review]
Nice?

**Pull request merged**